### PR TITLE
Add Get Centroid method

### PR DESCRIPTION
**Pull request by chuongmep**
### What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

### Description

This PR [adds/removes/fixes/replaces] this [feature/bug/etc]. 

### Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [x] 📓 docs
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: src/GShark/Geometry/Point3.cs [hunk 1]
```diff
@@ -496,6 +497,28 @@ public static Point3 PointBetween(Point3 p1, Point3 p2)
...
+        public static Point3 Centroid(IEnumerable<Point3> points)
+        {
+            return Centroid(points.ToArray());
+        }
+        /// <summary>
+        /// Returns the centroid of an arbitrary collection of points
+        /// </summary>
+        /// <param name="points">a list of points</param>
+        /// <returns>The centroid of the points</returns>
+        public static Point3 Centroid(params Point3[] points)
```
Comment (sonomirco) [inline src/GShark/Geometry/Point3.cs:515]: Could we have only one method which takes `IEnumerable<Point3>` or `IList<Point3>` as input?

File: src/GShark/Geometry/Point3.cs [hunk 2]
```diff
@@ -496,6 +497,28 @@ public static Point3 PointBetween(Point3 p1, Point3 p2)
...
+        /// Returns the centroid of an arbitrary collection of points
+        /// </summary>
+        /// <param name="points">a list of points</param>
+        /// <returns>The centroid of the points</returns>
+        public static Point3 Centroid(IEnumerable<Point3> points)
+        {
+            return Centroid(points.ToArray());
+        }
+        /// <summary>
+        /// Returns the centroid of an arbitrary collection of points
```
Comment (sonomirco) [inline src/GShark/Geometry/Point3.cs:511]: The point at the end of the sentence.

File: src/GShark/Geometry/Point3.cs [hunk 3]
```diff
@@ -496,6 +497,28 @@ public static Point3 PointBetween(Point3 p1, Point3 p2)
...
+        /// </summary>
+        /// <param name="points">a list of points</param>
```
Comment (sonomirco) [inline src/GShark/Geometry/Point3.cs:513]: A collection of points.

File: src/GShark/Geometry/Point3.cs [hunk 4]
```diff
@@ -496,6 +497,28 @@ public static Point3 PointBetween(Point3 p1, Point3 p2)
         {
             return Interpolate(p1, p2, 0.5);
         }
+        
+        /// <summary>
+        /// Returns the centroid of an arbitrary collection of points
```
Comment (sonomirco) [inline src/GShark/Geometry/Point3.cs:502]: Calculate the centroid of....

File: src/GShark.Test.XUnit/Geometry/Point3Tests.cs [hunk 5]
```diff
@@ -87,6 +87,19 @@ public void It_Divides_A_Point3d_By_A_Number()
             // Assert
             divisionResult.Equals(expectedPoint).Should().Be(true);
         }
+        
+        [Fact]
+        public void TestReturnCentroid()
```
Comment (sonomirco) [inline src/GShark.Test.XUnit/Geometry/Point3Tests.cs:92]: Could you please follow the same conventions we use for the unit test? Thanks.
Naming Convention and the AAA pattern (Arrange, Act and Assert)

**Review by sonomirco — CHANGES_REQUESTED** [review]

File: src/GShark/Geometry/Point3.cs (hunk 1 above)
Comment (chuongmep) [reply src/GShark/Geometry/Point3.cs]: will be change this, thank you.

File: src/GShark/Geometry/Point3.cs (hunk 2 above)
Comment (chuongmep) [reply src/GShark/Geometry/Point3.cs]: this mothod removed and merge with method input is IEnumerable

**Review by sonomirco — APPROVED** [review]

**Pull request merged**